rbac/migration_tool/migrate.py

```diff
@@ -26,109 +26,89 @@
 from migration_tool.utils import create_relationship, output_relationships
 
 from api.models import Tenant
-from .ingest import aggregate_v1_role
 
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
-BindingMappings = dict[str, dict[str, Any]]
-
 
 def get_kessel_relation_tuples(
-    v2_role_bindings: FrozenSet[V2rolebinding],
+    v2_role_bindings: Iterable[V2rolebinding],
     default_workspace: str,
-) -> tuple[list[common_pb2.Relationship], BindingMappings]:
+) -> list[common_pb2.Relationship]:
```

Comment:
The BindingMappings didn't need to be returned any more; by this point now we've already aggregated the same information.